en-GB/lang/numere.nlng

```diff
@@ -393,6 +396,7 @@ PARSERFUNCS_LISTFUNC_METHOD_ROWS_[DATA]=					rows
 PARSERFUNCS_LISTFUNC_METHOD_SCORROF_[DATA]=					scorrof(nCol1,nCol2,nRows1={1:nrows},nRows2={1:nrows})      VAL      - Calculates the Spearman correlation of the columns nCol1 and nCol2. The respective rows can be specified and correspond to the full dimension by default. The Spearman correlation can be calculated from all variable types. The calculation of the corresponding ranks is case sensitive
 PARSERFUNCS_LISTFUNC_METHOD_SIZE_[DATA]=					size                                                       {VAL}     - Returns the size of the table in rows and columns. Using the modificators .cols (length of columns), .rows (length of rows), .grid and .every(), the function may be restricted further
 PARSERFUNCS_LISTFUNC_METHOD_SPLT_[STRING]=					splt(cSep,keepEmpty=false)                                 {STR}     - Splits the string at the characters cSep in separate strings. With keepEmpty = true empty strings are returned as well, otherwise they are dropped
+PARSERFUNCS_LISTFUNC_METHOD_STARTSW_[STRING]=				startsw(sStr)                                               STR      - Gives the value True back, when the second Substring is found at the start of the first string. Otherwise, False.

```

Comment:
```suggestion
PARSERFUNCS_LISTFUNC_METHOD_STARTSW_[STRING]=				startsw(sStart)                                             LOG      - Returns true, if the string starts with sStart. Otherwise, false.
```